setup/setup_all.sh

```diff
@@ -39,7 +39,10 @@ setup_gcloud() {
 
 # Updating GCP Organizational policies
 update_gcp_org_policies() {
-  export ORGANIZATION_ID="$(gcloud organizations list --format='value(name)' | head -n 1)"
+  if [[ "$ORGANIZATION_ID" ==  "" ]]; then
+    export ORGANIZATION_ID="$(gcloud organizations list --format='value(name)' | head -n 1)"
```

Comment:
The organization id may not be the first entry

Reply:
We'll add more details in the README in later PR.